gammapy/datasets/tests/test_map.py

```diff
@@ -707,6 +707,61 @@ def test_map_fit(sky_model, geom, geom_etrue):
     assert not dataset_1.evaluators["test-model"].contributes
 
 
+@requires_data()
+@requires_dependency("ray")
+def test_map_fit_ray(sky_model, geom, geom_etrue):
+    from gammapy.datasets import DatasetsActor
+
+    dataset_1 = get_map_dataset(geom, geom_etrue, name="test-1")
+    dataset_2 = get_map_dataset(geom, geom_etrue, name="test-2")
+    datasets = Datasets([dataset_1, dataset_2])
+
+    models = Models(datasets.models)
+    models.insert(0, sky_model)
+
+    models["test-1-bkg"].spectral_model.norm.value = 0.5
+    models["test-model"].spatial_model.sigma.frozen = True
+
+    datasets.models = models
+    dataset_2.counts = dataset_2.npred()
+    dataset_1.counts = dataset_1.npred()
+
+    models["test-1-bkg"].spectral_model.norm.value = 0.49
+    models["test-2-bkg"].spectral_model.norm.value = 0.99
+
+    actors = DatasetsActor(datasets)
+    fit = Fit()
+    result = fit.run(datasets=actors)
```

Comment:
An other naive question: how this test check that a multiprocessing is really done? Is it possible to check that? Do we have a return message containing the number of processes that has been used? 
And should we add this information in the FitResult class?

Reply:
ray send messages about the number of core in use, I dont' think we need to add this info in the FitResult.